scripts/valhalla_build_config

```diff
@@ -596,7 +596,10 @@ def add_leaf_args(
             t = type(tree)
 
         arg = '--' + path.replace('_', '-').replace('\0', '-')
-        parser_.add_argument(arg, type=t, help=h, default=tree)
+        if t == list:
+            parser_.add_argument(arg, nargs="+", help=h, default=tree)
+        else:
+            parser_.add_argument(arg, type=t, help=h, default=tree)
```

Comment:
nit: a ternary would be nicer imho:
```suggestion
        parser_.add_argument(arg, nargs="+" if t == list else t, help=h, default=tree)
```

Reply:
@kevinkreiser That would be more concise, but I'm afraid this isn't going to work in this case. Your suggestion would replace the value of `nargs` with the result of the ternary operator. What I really want to do here is use the `nargs` parameter only if `t==list`, else set the paramter `type=t`.

Reply:
oh sorry i didnt look closely :smile: ignore the suggestion haha